packages/playground/src/components/account/account-register/account-register.tsx

```diff
@@ -33,9 +38,32 @@ export class AccountRegister {
     this.changeset[key] = event.target.value;
   }
 
-  formSubmitionHandler() {
-    console.log(this.changeset);
-    this.updateAccount(this.changeset);
+  async formSubmitionHandler() {
+    const data = this.changeset;
+
+    const httpResponse = await fetch(
+      "http://localhost:9000/api/create-account",
```

Comment:
env var?

Reply:
Yeah, sure

Reply:
@joelalejandro to be clear, i mean specifically for the host